tools/m2ctx.py

```diff
@@ -27,7 +27,7 @@ def get_c_file(directory):
 
 def import_c_file(in_file):
     in_file = os.path.relpath(in_file, root_dir)
-    cpp_command = ["gcc", "-E", "-P", "-Iinclude", "-Isrc", "-undef", "-D__sgi", "-D_LANGUAGE_C",
+    cpp_command = ["gcc", "-E", "-P", "-Iinclude", "-Isrc", "-undef", "-DM2CTX", "-D__sgi", "-D_LANGUAGE_C",
```

Comment:
If we are going to be touching m2ctx.py, perhaps we should just update it fully? (https://github.com/ethteck/m2ctx/)

Reply:
+ subrepo wouldnt be a bad idea

Reply:
I'm not really very interested in doing more in this PR, I changed just enough to get it to do what I wanted. A follow-up that properly updates it would be nice though if you're interested :eyes: